docs/source/api_ref_training.rst

```diff
@@ -76,6 +76,20 @@ Utilities to reduce memory consumption during training.
 
 .. _metric_logging_label:
 
+Schedulers
+-----------------
```

Comment:
the underline needs to match the length of the "Schedulers" title

Reply:
Fixed